### ISSUE TITLE
Cargo lock file out of date in 0.6.1

### ISSUE THREAD

**Issue by alerque** [soywod/himalaya#417]
Similar to #88, the lock file is out of sync in the 0.6.1 release tag.

**Comment by soywod** [comment]
My bad, I fix this ASAP. Sorry!

**Comment by soywod** [comment]
Strange because when I `cargo build` locally on `master` my `Cargo.lock` file does not change, how come? :thinking:

**Comment by alerque** [comment]
It does change when I run it. And after running it I can then run `cargo fetch --locked` which is what didn't work at first.

```console
$ cargo build
    Updating crates.io index
   Compiling toml v0.5.9
   Compiling serde_json v1.0.86
   Compiling erased-serde v0.3.23
   Compiling lettre v0.10.0-rc.7
   Compiling markup5ever v0.11.0
   Compiling html5ever v0.26.0
   Compiling ammonia v3.2.1
   Compiling himalaya-lib v0.4.0
   Compiling himalaya v0.6.1 (/home/caleb/projects/github/himalaya)
    Finished dev [unoptimized + debuginfo] target(s) in 14.20s
$ git diff
diff --git a/Cargo.lock b/Cargo.lock
index 5e42379..be2e16c 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -577,6 +577,8 @@ dependencies = [
 [[package]]
 name = "himalaya-lib"
 version = "0.4.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c6757df2ef0efa6418e90195dd3dcd2f8c144e374ebdc14b5967c7867a419394"
 dependencies = [
  "ammonia",
  "chrono",
```

**Comment by soywod** [comment]
I know why: locally I override the `himalaya-lib` in a way that it points to my local folder:

```toml
# ~/.cargo/config.toml

[patch.crates-io]
himalaya-lib = { path = "/path/to/my/himalaya-lib" }
```

Which leads to a different `Cargo.lock`. 
It is the first release using the lib, I did not see this issue coming back. I will publish a new minor release in the nearest days!

**Comment by soywod** [comment]
Sorry for the delay, this issue will be fixed by #433!
